### PR TITLE
feat: planning control evaluator

### PR DESCRIPTION
**Pull request by hayato-m126**
## Types of PR

- [x] New Features

## Description

- add planning/control evaluator node

### CODE REVIEW

File: docs/use_case/planning_control.en.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+# Evaluate Planning Control
+
+Evaluate whether Planning / Control metrics are output at specified times and conditions
+
+## Evaluation Method
+
+Launching the file executes the following steps:
+
+1. Execute launch of evaluation node (`planning_control_evaluator_node`), `logging_simulator.launch` file and `ros2 bag play` command
+2. Autoware receives sensor data output from input rosbag and the perception module performs recognition.
```
Comment (kosuke55) [inline docs/use_case/planning_control.en.md:10]: https://github.com/tier4/driving_log_replayer/pull/529
すみません、こちらと同様にrename予定です

File: docs/use_case/planning_control.en.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+3. Using the results of perception, Autoware output Metrics to `/diagnostic/planning_evaluator/metrics` for planning and `/diagnostic/control_evaluator/metrics` for control.
```
Comment (kosuke55) [inline docs/use_case/planning_control.en.md:11]: tier4/driving_log_replayer#529
すみません、こちらと同様にrename予定です

Comment (kosuke55) [reply docs/use_case/planning_control.en.md:11]: https://github.com/autowarefoundation/autoware.universe/pull/8152

Comment (hayato-m126) [reply docs/use_case/planning_control.en.md]: I updated topic name in f87670d0455898e9a2da56d03dd80073479ca28b

**Review by kyoichi-sugahara — APPROVED** [review]
I confirmed that this feature works in my environment and left evidence for that in following jira's ticket.
[TIER INTERNAL LINK](https://tier4.atlassian.net/browse/RT1-7301) 
Thank you so much for your work :heart_eyes:

**Pull request merged**